spyder/widgets/fileswitcher.py

```diff
@@ -663,10 +663,12 @@ def setup_file_list(self, filter_text, current_path):
                     text_item += " [{0:} {1:}]".format(self.line_count[index],
                                                        _("lines"))
                 if max_width > self.list.width():
-                    text_item += u"<br><i style='color:{0:}'>{1:}</i>".format(
+                    text_item += (u" <i style='color:rgb(153,153,153)'>"
+                                  " {1:}</i>").format(
                             ima.MAIN_FG_COLOR, short_paths[index])
                 else:
-                    text_item += u"<br><i style='color:{0:}'>{1:}</i>".format(
+                    text_item += (u" <i style='color:rgb(153,153,153)'> "
```

Comment:
Please use here the new `PATH_FG_COLOR` constant in a similar way as explained above